### PR TITLE
add ogles_gpgpu v0.2.5

### PR DESCRIPTION
**Pull request by headupinclouds**

### CODE REVIEW

File: cmake/projects/ogles_gpgpu/hunter.cmake [hunk 1]
```diff
@@ -117,7 +117,18 @@ hunter_add_version(
...
     SHA1
     ce38c35533fbb8f8efc30eb74d300598ee2132aa
-)
+    )
+
+hunter_add_version(
+    PACKAGE_NAME
+    ogles_gpgpu
+    VERSION
+    0.2.5
```
Comment (ruslo) [inline cmake/projects/ogles_gpgpu/hunter.cmake]: Please add it to `cmake/configs/default.cmake`

Comment (headupinclouds) [reply cmake/projects/ogles_gpgpu/hunter.cmake]: Oops.  Done.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2587
* https://travis-ci.org/ingenue/hunter/builds/290501294?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2589
* https://travis-ci.org/ingenue/hunter/builds/290670871?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.129

**Pull request merged**